app/controllers/doctors_controller.rb

```diff
@@ -30,28 +31,34 @@ def edit
 end
 
 def update
+  @doctor = Doctor.find(params[:id])
+  @doctor.update(doctor_params)
+  redirect_to "doctors/index"
```

Comment:
Change redirect path to prefix paths generated by rails (`doctors_path`) to give flexibility in changing actual index route in routes.rb